### DISCUSSION TITLE
posts/Go/go-kit-tutorial-04/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#294] [Announcements]
https://www.liwenzhou.com/posts/Go/go-kit-tutorial-04/ 

 李文周的Blog提供免费的全套Go语言学习教程。本文主要介绍了go-kit的教程的第四篇，介绍了如何使用go-kit中间件的概念，并介绍go kit如何使用日志，以及go kit如何集成zap日志

**Comment by gotosre** [comment]
添加中间件时，包含service接口，再把service接口实现一遍添加亿些细节，好不习惯。。。

**Comment by mkshift** [comment]
请问这个有完全一些的示例源码么

**Comment by 00knight** [comment]
还有人吗
